pom.xml

```diff
@@ -197,6 +197,9 @@
             </goals>
             <configuration>
               <createSourcesJar>true</createSourcesJar>
+              <transformers>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
```

Comment:
Yes we do as Jetty use ServiceLoader to load httpfieldencoder (header parsing). There are different META-INF/services which need to be merged. Otherwise http1 or http2 will not work. This one https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/jetty-http/src/main/resources/META-INF/services/org.eclipse.jetty.http.HttpFieldPreEncoder and this one https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/jetty-http2/http2-hpack/src/main/resources/META-INF/services/org.eclipse.jetty.http.HttpFieldPreEncoder .